test/browser/webpack.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       },
       {
         test: /sinon(\\|\/)pkg(\\|\/)sinon\.js/,
-        loader: 'imports?define=>false,require=>false'
+        loader: 'imports-loader?define=>false,require=>false'
```

Comment:
None of the crazy webpack/sinon workarounds are necessary as of sinon v2! You should be able to safely delete this rule and remove `imports-loader` from this project